### PR TITLE
Initial draft for running Discourse through cf

### PR DESCRIPTION
**Pull request by JJediny**
@dlapiduz

### CODE REVIEW

File: content/apps/discourse.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  vcap = ENV["VCAP_SERVICES"]
+  JSON.parse(vcap)["user-provided"].each do |service|
+    if service["name"].include?("-env")
+      service["credentials"].each do |k,v|
+        ENV[k] = v
+      end
+    end
+  end
+rescue
+  puts "Error loading env vars"
```
Comment (michael-balint) [inline content/apps/discourse.md:49]: needs an `end` statement

File: content/apps/discourse.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  - community-redis-swarm
+  - community-db
+  - discourse-env
+```
+## Create Services in CF Marketplace
+Use the redis and postgres services available within cloud.gov cf marketplace to create two new service instances. Replace the names of the newly minted services into the manifest files
+
+## Create User Provided Service for VCAP credentials
+```
+cf cups discource-env -p '{"username":"admin","password":"pa55woRD"}'
```
Comment (michael-balint) [inline content/apps/discourse.md:88]: typo: `discource-env` should be `discourse-env`

**Comment by afeld** [comment]
Since this isn't totally cloud.gov-specific, would it make sense to put these instructions in https://github.com/18F/how-to-deploy instead?

**Comment by afeld** [comment]
Closing per my comment above.

**Pull request closed without merging**